_posts/2017-11-30-utah-sgid-statewide-roads-data-layer-updates-11302017.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: post
+status: publish
+published: true
+title: Utah SGID Statewide Roads Data Layer Updates 11/30/2017
+author:
+  display_name: Greg Bunce
+  email: gbunce@utah.gov
+date: 2017-11-30
+categories:
+  - Featured
+tags:
+  - centerlines
+  - data
+  - gis
+  - roads
+  - sgid
+  - statewide
+  - utah
+---
+
+Updates were recently made to the SGID10.Transportation.Roads and [SGID10.Transportation.RoadsODM]({{ site.baseurl }}{% post_url 2017-08-15-major-updates-coming-to-roads-data-model %}) feature classes that resides on the [Utah SGID ArcSDE database server]({{ "/sgid-database/" | prepend: site.baseurl }}).
```

Comment:
"resides" should be "reside"

Reply:
good catch - thx.  i made the change in my repo and committed.  is that the best way to make that change?

Reply:
Yep. Perfect.